pom.xml

```diff
@@ -44,18 +56,6 @@
     <version.org.hawkular.bus>0.3.3.Final</version.org.hawkular.bus>
   </properties>
 
-  <modules>
-    <module>api</module>
-    <module>hawkular-inventory-json-helper</module>
-    <module>impl-tinkerpop-parent</module>
-    <module>hawkular-inventory-cdi</module>
-    <module>rest-servlet</module>
-    <module>bus</module>
-    <module>bus-api</module>
-    <module>hawkular-integrated-inventory-rest</module>
```

Comment:
Yes, that's expected. Changes will perhaps also be needed in Agent. IMO this is a reason to release inventory ASAP after merging this and change the dep names in downstream.